pkg/markdown/markdown.go

```diff
@@ -10,11 +13,16 @@ func WithoutIndentation() glamour.TermRendererOption {
 }
 
 // WithWrap is a rendering option that set the character limit for soft
-// wrapping the markdown rendering. There is a max limit of 120 characters.
+// wrapping the markdown rendering. There is a max limit of 120 characters,
+// unless the user overrides with an environment variable.
 // If 0 is passed then wrapping is disabled.
 func WithWrap(w int) glamour.TermRendererOption {
-	if w > 120 {
-		w = 120
+	width, err := strconv.Atoi(os.Getenv("GH_MDWIDTH"))
+	if err != nil {
+		width = 120
+	}
```

Comment:
If I understand this correctly, if the `GH_MDWIDTH` variable isn't set this will return an error and set the default width to 120. Should we be a bit more explicit, here, for failures? What if the user tried to set the variable and set it incorrectly (i.e. not a number)? Should we let them know?

I'm on the fence about it, so looking for your thoughts.

Reply:
The observed behavior is:

- not set: default value
- random text: default value
- negative number: gets treated like 0
- positive number: sets the default width

To me that's totally fine and like I would expect.  I don't see any need to inform the user about anything, the mismatch in their expectation vs reality will tell them they used the variable incorrectly.  But the correct usage is fairly obvious and if the user is using the variable at all they probably know to use an integer (it's also in the help text).

The code should handle bogus inputs for safety reasons, of course, but seems to already have the correct behavior in this regard.

Let me know if you concur.  Thanks for reviewing.